lib/documents/schemas/international_development_funds.json

```diff
@@ -142,10 +142,6 @@
           "value": "burkina-faso",
           "label": "Burkina Faso"
         },
-        {
-          "value": "burma",
```

Comment:
I don't know enough about this repo to know if removing `burma` will cause regression issues (e.g. if there are things that rely on the `burma` value existing).

Reply:
I'm not sure either on the downstream implications of this file - but in the case of Burma it has recently been renamed to Myanmar across the rest of gov.uk by Platform Health so I think this change at least should be OK.

Reply:
https://trello.com/c/8Y2fbezn/879-country-name-change-burma-myanmar

Reply:
Good catch!

We might need to check every published document of type `international_development_fund` and change any values of `burma` to `myanmar` before doing this, otherwise I suspect the finder will change to filter against Myanmar, but they'll be no documents tagged to it.